verification/curator-service/ui/src/components/App.tsx

```diff
@@ -313,7 +320,8 @@ export default function App(): JSX.Element {
             text: 'Home',
```

Comment:
WDYT about calling this charts and making it visible to curators and admins? And always having the home page be the linelist page? Doesn't seem too useful of a homepage for non-readers either. We could then add a different homepage that is useful to everyone (pending UX).

I think this current logic also shows this home page for not-logged in users which we don't want as it's empty.

Reply:
Done.

- FYI logged out users currently see this
![image](https://user-images.githubusercontent.com/1255432/93565160-2c772e80-f98b-11ea-9452-b8cda6444aec.png)

not super useful but once we have an about page (/terms) seem a bit too "harsh" as a front page I think it'd be a good candidate.

- readers only go to linelist
- curators end up on the charts yes which is the "Home" page, so in that sense it makes sense to rename it to charts pending a proper home page from UX.

Reply:
Cool, thanks. Filed #1133 for design of a simple home page.